<launch>
  <node name="collision_detection" pkg="ros_collision_detection" type="collision_detection" output="screen"/>
  <param name="ttc_algorithm_classname" value="CircleAlgorithm" type="str" />
</launch>